### ISSUE TITLE
An error occurs when trying to open one dialog sometimes

### ISSUE THREAD

**Issue by v-xianya** [microsoft/AzureStorageExplorer#5674]
**Storage Explorer Version**: 1.24.0
**Build Number**: 20220427.2
**Branch**: rel/1.24.0
**Platform/OS**: MacOS Monterey 12.3 (Apple M1 Pro)
**Architecture** x64
**How Found**: From running test case
**Regression From**: Not a regression

## Bug Descriptions ##
1. An error occurs when trying to open one dialog sometimes.
2. The error doesn't occur when opening the dialog again.

## Steps to Reproduce ##
1. Expand one storage account -> Tables.
2. Create a table -> Click 'Add Entity'.
3. Check no error occurs.

## Expected Experience ##
No error occurs.

## Actual Experience ##
An error occurs sometiems.
<img width="271" alt="Screen Shot 2022-04-26 at 23 40 46" src="https://user-images.githubusercontent.com/41351993/165729454-3bc12f1a-d856-493d-9375-a942a9f4c577.png">

## Additional Context ##
When trying to open 'Clone and rehydrated' dialog. The below error occurs:
<img width="505" alt="Screen Shot 2022-04-26 at 19 14 58" src="https://user-images.githubusercontent.com/41351993/165729939-92390896-0547-41dc-a3f8-12052dcacfde.png">

**Comment by MRayermannMSFT** [comment]
@v-xianya so is this something y'all see in 1.23.1 or any other old versions?

**Comment by v-xianya** [comment]
@MRayermannMSFT 
yes, I tried to reproduce this issue on 1.23.1. and it can be reproduced.

Not tried on other old versions.

**Comment by MRayermannMSFT** [comment]
@v-xianya thank you for confirming that. Can you try some other older versions? How about:
- 1.22
- 1.20
- 1.18

We'll keep trying to reproduce on our own as well.

**Comment by v-xianya** [comment]
@MRayermannMSFT I tried to reproduce this issue on 1.22.0 today. It can not be reproduced. 
Will continue try to reproduce this issue on 1.22.0 and other older versions if we meet this issue on the latest build again.

**Comment by v-xianya** [comment]
When clicking 'Change Access Tier...' against multi selections, there is an error dialog.
<img width="708" alt="Screen Shot 2022-05-10 at 19 14 06" src="https://user-images.githubusercontent.com/41351993/167755885-f09967b0-a606-484b-bfe4-e9790f8a48af.png">

**Comment by v-xianya** [comment]
Fail to get properties for one queue with the below error.
<img width="1145" alt="Screen Shot 2022-05-10 at 20 16 25" src="https://user-images.githubusercontent.com/41351993/167762204-7b5fee15-0b00-4375-8e0f-92ee281dead4.png">

**Comment by MRayermannMSFT** [comment]
@v-xianya have you ever seen this with the connect dialog? Once it happens, are all dialogs broken afterwards?

**Comment by v-xianya** [comment]
@MRayermannMSF I haven't seen this issue with the connect dialog. we'll check whether all dialogs are broken once this happens.
